Manually configurable minimum protocol version (#14054)
minetest/minetest: builtin/settingtypes.txt

#    to new servers, but they may not support all new features that you are expecting.
strict_protocol_version_checking (Strict protocol checking) bool false

#    Define the oldest clients allowed to connect.
#    Older clients are compatible in the sense that they will not crash when connecting
#    to new servers, but they may not support all new features that you are expecting.

> #    This allows more fine-grained control than strict_protocol_version_checking.
> #    Minetest may still enforce its own internal minimum, and enabling
> #    strict_protocol_version_checking will effectively override this.

Reviewer: ```suggestion
#    This allows for more fine-grained control than strict_protocol_version_checking.
#    Minetest still has its own internal minimum, and enabling
#    strict_protocol_version_checking will effectively override this.
```
